app/assets/javascripts/components/faye_messages.jsx

```diff
@@ -0,0 +1,41 @@
+class FayeMessages extends React.Component {
+  constructor(props) {
+    super(props);
+     this.state = {
+      urlType:`/assets/chatroom.jpg`
+      };
+  }
+
+  generate_private_chat(e) {
+    var name = e.target.innerHTML;
+    if (!document.querySelector(`#${name}`)) {
+      document.getElementById('message').value = '@'+name+' (private message): ';
+    }
+  };
+
+  generate_users_online(user_list) {
```

Comment:
the same comment that above + attributes should be as well camelCase not user_list but userList